### ISSUE TITLE
Boot pxe

### ISSUE THREAD

**Issue by renatogfilhinho** [FOGProject/fogproject#596]
I have three machines, for example DP35DP and an Inspiron 5548 that cannot access Fog via pxe, but they boot via LTSP, can anyone help me? Thanks

**Comment by darksidemilk** [comment]
Do you have other machines that boot fine?
What version of fog?

**Comment by renatogfilhinho** [comment]
Yes, I have, I even created an image of it using Fog and in addition to it, I have vms in virtualbox that also find Fog. Thanks for answering.

**Comment by renatogfilhinho** [comment]
I use dnsmasq as a proxy and everything works fine, I use undionly.kpxe, but I've already tried undionly.pxe, undionly.kkpxe and these machines even log into dnsmasq, but I get the message "Error: PXE-E74 bad or missing PXE menu and or prompt information"

**Comment by darksidemilk** [comment]
I found this on that specific error https://forums.ivanti.com/s/article/Error-PXE-E74-bad-or-missing-PXE-menu-and-or-prompt-information?language=en_US

Is the bios updated?
Do you have csm configured to use the bios versions of pxe.
Do these machines have an option for efi?

**Comment by renatogfilhinho** [comment]
Thanks for your answers! I had forgotten to answer the FOG version, it is 1.5.10, stable.

"I found this on that specific error https://forums.ivanti.com/s/article/Error-PXE-E74-bad-or-missing-PXE-menu-and-or-prompt-information?language=en_US"

I had already visited this site, it didn't help me, thank you.

"Is the bios updated?"

No, the cards are old, Intel no longer supports them, and has even deleted any downloads of them.

"Do you have csm configured to use the bios versions of pxe."
No, I dont have any csm configured

"Do these machines have an option for efi?"
No, they don't.

The most curious thing is that they all boot from LTSP, the network card works normally.

**Comment by darksidemilk** [comment]
Are you booting to LTSP via pxe? so you have 2 pxe servers at the moment?

Are you using Linux or windows DHCP and is it pointing to the LTSP or FOG for the pxe server with the bootfile?

**Comment by renatogfilhinho** [comment]
Yes, machines that don't boot in FOG, boot in LTSP. I only have one PXE server in production, which is LTSP. FOG is on a completely separate, segregated network. I use dnsmasq in both cases as a proxy.

**Comment by darksidemilk** [comment]
But these machines with the PXE error are on which network? 
Do you have access to your DHCP server?

**Comment by renatogfilhinho** [comment]
LTSP uses dnsmasq as a proxy on network A (production), FOG uses dnsmasq on network B (lab). Machines that have problems on network B (FOG) do not have problems when connected to LTSP (network A). The FOG network (network B) and the FOG server itself are functional, I even have physical and virtual stations that can access the FOG menu via PXE.

**Comment by renatogfilhinho** [comment]
The router is a d-link and I have access to it.

**Comment by renatogfilhinho** [comment]
[log-dnsmasq.txt](https://github.com/user-attachments/files/15991471/log-dnsmasq.txt)

Below is the partial log of a machine that does not access FOG and one that does.

**Comment by renatogfilhinho** [comment]
And to avoid doubts, I formatted the server, reinstalled Debian 12.5 and installed FOG stable 1.5.10 again, however, these computers still do not boot from FOG via PXE.

More info ...

computers ok - vm virtualbox (vm running inside the Inspiron 5548), Intel dh55tc
computers not ok - Inspiron 5548, intel dp35dp, Intel dg965wh

**Comment by darksidemilk** [comment]
This does seem specific to the models, I'm not 100% sure what's going on. It could be an adapter driver kernel issue since it works in a VM on the machine (which would use a different net adapter technically) but not the actual machine.

I found this forum post that might be related to your situation https://forums.fogproject.org/topic/9959/dell-inspiron-15-7559/3?_=1719505597154

There are also some other various posts in the forum if you search 'ltsp' that talk about going from fog to LTSP which I imagine you could reverse for your other issue.

**Comment by mastacontrola** [comment]
Can you post images of the error you're seeing with these machines pxe booting to FOG?

Just saying things and us guessing as to potential scenarios is relatively useless at this point in time.

While I believe you are having issues, just trying to figure out with WAGs at this point is not very helpful.

Can you help me help you with this?

We don't support or even know about the software LTSP. While I'm glad that it is booting for you, I'd much rather fix FOGs stuff to help get things working.

Note, this doesn't mean LTSP couldn't potentially be built to have a FOG tie in, but when you chainload things you're passing information from one bit to another. The "another" things necessary in our builds of ipxe would also need to be compiled in LTSP, which I cannot guarantee is the case. So if we can see what's happening and causing issues, it's pretty likely we can tell you "do this and this, or can you try that".

**Comment by renatogfilhinho** [comment]
It was my mistake in mentioning the LTSP, just forget it. The case is that the machines that I used the models above are very old, approximately from 2008, so the onboard network card firmware may not support some FOG functionality. To remedy this, I created a bootable pendrive with the ipxe.iso iso and that worked. @mastacontrola, thanks for your attention, but all the logs are above, all the pertinent information too, I don't know how I can help you help me, but thank you. Problem solved, I thank everyone for the extreme attention and time given to help me, long live the FOG community!
